src/SFA.DAS.EmployerAccounts/Queries/RemovePayeFromAccount/RemovePayeFromAccountCommandValidator.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.Threading.Tasks;
 using SFA.DAS.Authorization;
```

Comment:
looks like this using statement has been removed in some but not all cases for these classes

Reply:
You are correct, I think I've missed the removal in quite a few places.  If I get time I'll clear them up.